operator/internal/manifests/config.go

```diff
@@ -79,6 +79,11 @@ func ConfigOptions(opt Options) config.Options {
 		Stack:     opt.Stack,
 		Namespace: opt.Namespace,
 		Name:      opt.Name,
+		Compactor: config.Address{
+			FQDN:     fqdn(NewCompactorHTTPService(opt).GetName(), opt.Namespace),
+			Port:     grpcPort,
```

Comment:
Sorry, I did not spot this the first time.

```suggestion
			Port:     httpPort,
```